package.json

```diff
@@ -38,5 +38,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "cors": "^2.8.5",
+    "express": "^4.18.2"
```

Comment:
We shouldn't need these packages on the frontend. Can you remove them or explain why they're necessary.

Reply:
These were added while I was trying some stuff to make the login, I forgot that they were added.

Reply:
These still need to be deleted.